packages/cubejs-api-gateway/index.js

```diff
@@ -195,7 +195,7 @@ class ApiGateway {
   initApp(app) {
     app.get(`${this.basePath}/v1/load`, this.checkAuthMiddleware, (async (req, res) => {
       try {
-        const query = JSON.parse(req.query.query);
+        const query = await Joi.validate(JSON.parse(req.query.query),querySchema)
```

Comment:
I believe this one is redundant as it called by `normalizeQuery`?

Reply:
ya right . Thanks